python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -125,15 +124,15 @@ class DcrAssembleCoaddConfig(CompareWarpAssembleCoaddConfig):
         default=2,
         doc="Minimum radius of a region to include in regularization, in pixels."
     )
-    imageWarpMethod = pexConfig.Field(
-        dtype=str,
-        doc="Name of the warping kernel to use for shifting the image and variance planes.",
-        default="lanczos3",
+    imageInterpOrder = pexConfig.Field(
+        dtype=int,
+        doc="The order of the spline interpolation used to shift the image plane.",
+        default=3,
     )
-    maskWarpMethod = pexConfig.Field(
-        dtype=str,
-        doc="Name of the warping kernel to use for shifting the mask plane.",
-        default="bilinear",
+    maskInterpOrder = pexConfig.Field(
+        dtype=int,
+        doc="The order of the spline interpolation used to shift the mask plane",
+        default=1,
```

Comment:
I've removed `maskInterpOrder` entirely. The name and description were misleading, since it was actually used for the weights, not the mask plane. And, the weights really need to use the same interpolation as the image, so it shouldn't be configurable by itself.